package.json

```diff
@@ -5,8 +5,10 @@
   "main": "server.js",
   "scripts": {
     "start": "node src/server.js",
-    "dev": "nodemon src/server.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "dev": "nodemon src/server.js && npx sequelize-cli db:migrate",
```

Comment:
@bright258 migration command should be before the server command

Reply:
Done